src/pages/speakers/speakers-tips/index.astro

```diff
@@ -7,27 +7,27 @@ import {Image} from "astro:assets"
 <Layout title="JavaZone 2024 - Speakers - Submission tips">
     <div class="grid md:inline-block">
         <h1 class="text-[5.5vw] mx-auto md:text-[4vw]">Submission tips for speakers</h1>
-        <div class="text-[4vw] mx-auto md:text-[2.5vw]">
+        <div class="text-[4vw] mx-auto md:text-[2vw]">
             <ButtonLink href="https://talks.javazone.no/" target="_blank">Submit your talk!</ButtonLink>
         </div>
-        <h2 class="text-[4vw] md:text-[2.5vw]">Increase the chance of your talk being accepted</h2>
-        <p class="text-[3.5vw]  md:text-[2.5vw]">
+        <h2 class="text-[4.5vw] md:text-[2vw]">Increase the chance of your talk being accepted</h2>
+        <p class="text-[3.5vw]  md:text-[2vw]">
             There is generally a very high correlation between how much work one puts into the
             creation of a proposal and the chance that the proposal is accepted. Even a proposal with
             an extremely cool theme, entertaining title, and an experienced speaker may be excluded if
             the description is lacking.
         </p>
         <div class="mt-4 p-4 bg-white rounded-2xl border-black border-2 text-center">
-            <h3 class="text-[3.5vw] md:text-[2.5vw]">Things to think about when writing your abstract</h3>
+            <h3 class="text-[3.5vw] md:text-[2vw]">Things to think about when writing your abstract</h3>
             <p class="font-bold text-[3vw]  md:text-[2.2vw]">We have gathered some tips that should be considered when
                 writing a JavaZone proposal, and a few reasons why proposals may be rejected.</p>
         </div>
         <div class="md:flex mt-4">
             <section
                 class="w-auto md:w-[70%] md:-mx-0 rounded-2xl border-black border-2 bg-white [&&]:px-[1rem] flex gap-4">
                 <article class="w-[50%] h-auto bg-[#00f3d2] p-[1rem] border-2 border-black rounded-2xl">
-                    <h3 class="text-[4vw] md:text-[2.5vw]">Create a good title</h3>
-                    <p class="text-[3.5vw] md:text-[2vw]">
+                    <h3 class="text-[4.5vw] md:text-[2vw]">Create a good title</h3>
```

Comment:
Hvorfor definerer vi størrelse av tekst i p og h for hver heading og paragraph? Burde ikke sånne ting være definert for alle headere og paragrapher globalt i en felles CSS fil? @tanettrimas @Ellaoui94 
Dette her ligner på `style `param mer enn `class`. Jeg kan se for meg at det er lett å ta feil hvis man må justere størrelser i 30-40 steder hver gang.